docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -298,6 +301,91 @@ To log out of the aws application and remove credentials:
 $ tsh app logout awsconsole-test
 ```
 
+## Step 9/9. Access applications using AWS SDKs
+
+First, log into the previously configured console app if you haven't already
+done so.
+
+```code
+$ tsh app login --aws-role ExamplePowerUser awsconsole-test
+```
+
+To connect an application that uses AWS SDKs, you can start either a local
+HTTPS proxy or a local AWS Service Endpoint proxy.
+
+<Tabs>
+  <TabItem label="HTTPS proxy">
+  By default, starting the AWS app proxy creates a local HTTPS proxy server
+  that forwards AWS requests to the Teleport Proxy Service, enabling you to
+  access AWS applications.
+
+  Now, use the following command to start the proxy your applications will be
+  connecting to:
+
+  ```code
+  $ tsh proxy aws -p 23456
+  Started AWS proxy on http://127.0.0.1:23456.
+
+  Use the following credentials and HTTPS proxy setting to connect to the proxy:
+    AWS_ACCESS_KEY_ID=<access-key-id>
+    AWS_SECRET_ACCESS_KEY=<secret-access-key>
+    AWS_CA_BUNDLE=<ca-bundle-path>
+    HTTPS_PROXY=http://127.0.0.1:23456
+  ```
+
+  Use the displayed AWS credentials and HTTPS proxy settings when configuring
+  your application.
+
+  It is important to check how AWS credentials and HTTPS proxy setting can be
+  configured for your application. For example, many command line tools like
+  `terraform` or `eksctl` supports setting the above AWS credentials and the
+  HTTPS proxy using environment variables:
+
+  ```code
+  $ export AWS_ACCESS_KEY_ID=<access-key-id>
+  $ export AWS_SECRET_ACCESS_KEY=<secret-access-key>
+  $ export AWS_CA_BUNDLE=<ca-bundle-path>
+  $ export HTTPS_PROXY=http://127.0.0.1:23456
+  $ terraform
```

Comment:
terraform just print to help. What do you think about using a real terraform command like `terraform apply ` or `terraform plan` ? 
```suggestion
  $ terraform plan
```

Reply:
good catch. I meant `terraform plan` but not sure how I missed it. Thanks!